lib/redux/slices/drawer.ts

```diff
@@ -0,0 +1,25 @@
+import { createSlice } from '@reduxjs/toolkit'
+import { DrawerComponentProps } from '@/utils/types/redux'
+
+const initialState: DrawerComponentProps = {
+    drawerDisplay: '',
+    showDrawer: false
```

Comment:
You never updated this to be `content`. While you're at it, change the other one to be `showing`